src/components/capture/create/index.tsx

```diff
@@ -135,130 +134,108 @@ function CaptureCreation() {
-                .then(() => {
+        discovers: () => {
+            return waitFor.base(
+                supabaseClient.from(TABLES.DISCOVERS),
+                (payload: any) => {
+                    setDraftId(payload.new.draft_id);
+                },
+                'captureCreation.test.failedErrorTitle'
+            );
+        },
+        publications: () => {
+            return waitFor.base(
+                supabaseClient.from(TABLES.PUBLICATIONS),
+                () => {
                     setFormState({
                         status: CaptureCreationFormStatus.IDLE,
+                        exitWhenLogsClose: true,
+                        saveStatus: intl.formatMessage({
+                            id: 'captureCreation.status.success',
+                        }),
                     });
-                })
-                .catch(() => {});
-        },
-        waitForFinish: () => {
-            cleanUpEditor();
-            resetFormState(CaptureCreationFormStatus.SAVING);
-            const draftsSubscription = supabaseClient
-                .from(TABLES.DRAFTS)
-                .on('*', async (payload) => {
-                    if (payload.new.job_status.type !== 'queued') {
-                        if (payload.new.job_status.type === 'success') {
-                            setFormState({
-                                status: CaptureCreationFormStatus.IDLE,
-                                exitWhenLogsClose: true,
-                                saveStatus: intl.formatMessage({
-                                    id: 'captureCreation.status.success',
-                                }),
-                            });
-                            const notification: Notification = {
-                                description:
-                                    'Your new capture is published and ready to be used.',
-                                severity: 'success',
-                                title: 'New Capture Created',
-                            };
-                            showNotification(notification);
-                        } else {
-                            setFormState({
-                                error: {
-                                    title: 'captureCreation.save.failedErrorTitle',
-                                },
-                                saveStatus: intl.formatMessage({
-                                    id: 'captureCreation.status.failed',
-                                }),
-                            });
-                        }
-
-                        await drafts.done(draftsSubscription);
-                    }
-                })
-                .subscribe();
 
-            return draftsSubscription;
+                    showNotification(notification);
+                },
+                'captureCreation.save.failedErrorTitle'
+            );
         },
     };
 
-    const prepareCatalogForSaving = () => {
```

Comment:
We update the `draft_spec` table now and no longer need to build up a catalog to send.